Automattic/jetpack
# RNA: the Observer package

#### Changes proposed in this Pull Request:
- Introduce the RNA Observer package for event tracking.
- Modify the RNA Connection package to fire certain events: `CONNECTION_SITE_CONNECTED`, `CONNECTION_DISCONNECTED`.
- Modify the Backup plugin to switch from the old callback approach to using the Observer events.

#### Jetpack product discussion
p9dueE-3ud-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to the Backup plugin, connect if disconnected.
2. Click "Disconnect", the Disconnect modal should pop up.
3. Click "Disconnect" in the modal, the "successfully disconnected" modal should pop up.
4. Click "Back to WordPress", the Connection Screen should appear instead of the Backup dashboard.
